DigitalLearningSolutions.Data/Models/User/Delegate.cs

```diff
@@ -13,8 +13,8 @@ public class Delegate
             UserCentreDetails = userCentreDetails;
         }
 
-        public DelegateAccount DelegateAccount { get; set; }
-        public UserAccount UserAccount { get; set; }
-        public UserCentreDetails? UserCentreDetails { get; set; }
+        public DelegateAccount DelegateAccount { get; }
+        public UserAccount UserAccount { get; }
+        public UserCentreDetails? UserCentreDetails { get; }

```

Comment:
I have now been informed that these should be changed to, for e.g., `public readonly DelegateAccount DelegateAccount; `, like [this](https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/DigitalLearningSolutions.Web/ViewModels/LearningPortal/Available/AvailableCourseViewModel.cs#L10).

Reply:
I think there is some mild benefit in having `readonly` instead of just not having a `set` accessor (quicker to recognise).
But I don't think it's a massive deal. I mentioned it to Simon because
1. I didn't realise how widely we already use it.
2. Simon's new to C# so you might not be aware of the keyword.

Reply:
I'd argue convention in the project is to have get only, 487 vs 119. 
Personally I find the get only properties slightly more recognizable, especially in places where there may be other static/private/const fields that would be mixed in with the public readonly fields.
It also takes away the references count in VS when you convert to a readonly field
![image](https://user-images.githubusercontent.com/59561751/174022703-a72c3830-dbc5-45be-8708-2a31040b827b.png)

Reply:
Interesting, never noticed that. Grand, feel like that outweighs my readbility preference, so let's stick with {get;} instead then.